packages/mobile/src/components/ShortAccountLink.tsx

```diff
@@ -0,0 +1,57 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import Clipboard from '@react-native-community/clipboard'
+import React from 'react'
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import CopyIcon from 'src/icons/CopyIcon'
+import Logger from 'src/utils/Logger'
+
+interface Props {
+  shortLink: string
+  touchDisabled?: boolean
+}
+
+export default function ShortAccountLink({ shortLink, touchDisabled }: Props) {
+  const onPressLink = () => {
+    if (!shortLink.length) {
+      return
+    }
+    Clipboard.setString(shortLink)
+    Logger.showMessage('Short link copied')
+  }
+  // Remove 'https://' from displayed link
+  const shorterLink = shortLink.slice(8)
+  const formattedLink = (
+    <View style={[styles.line]}>
+      <Text style={[styles.text]}>{shorterLink}</Text>
+      <CopyIcon />
+    </View>
+  )
+
+  return touchDisabled ? (
+    <View style={styles.container}>{formattedLink}</View>
+  ) : (
+    <TouchableOpacity style={styles.container} onLongPress={onPressLink} onPress={onPressLink}>
+      {formattedLink}
+    </TouchableOpacity>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: 200,
```

Comment:
I don't see any reason to hard code the width like this. How about removing this `width: 200` and the one in `line` and making the `horizontalPadding` a bit larger? (15?)

Right now all codes have 4 digits but as these get used up we'll eventually start seeing codes with more digits so it's better to be prepared for that.